ctms/acoustic_sync.py

```diff
@@ -1,10 +1,48 @@
 #!/usr/bin/env python3
 """Run continuously in the background, syncing acoustic with our db."""
 
+from datetime import datetime, timezone
 from time import monotonic, sleep
+from typing import List
 
 from ctms import config
+from ctms.crud import (
+    delete_acoustic_record,
+    get_acoustic_record_as_contact,
+    get_all_acoustic_records_before,
+    retry_acoustic_record,
+)
 from ctms.database import get_db_engine
+from ctms.models import PendingAcousticRecord
+from ctms.schemas import ContactSchema
+
+
+def sync_contact_with_acoustic(contact: ContactSchema):
+    print("sync here")
+    is_success = True
+    # TODO:
+    #  Need to define what a successful response from Acoustic would be, and make the request.
+    #   Upon success, the record is discarded;
+    #   Upon failure, the record is up for retry
+    return is_success
+
+
+def sync_pending_record(db, pending_record: PendingAcousticRecord):
+    contact: ContactSchema = get_acoustic_record_as_contact(db, pending_record)
+    is_success = sync_contact_with_acoustic(contact)
+
+    if is_success:
+        delete_acoustic_record(db, pending_record)
+    else:
+        retry_acoustic_record(db, pending_record)
+
+
+def sync_records(db):
```

Comment:
calling this method at this print:
https://github.com/mozilla-it/ctms-api/pull/188/files#diff-56721425dd97d7b4cd2f46cf938b7857447b6033e53803d90b5bb8bf1bc0fd6fR51